frontend/src/metabase/search/components/SidebarFilter/SidebarFilter.unit.spec.tsx

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable react/prop-types */
+import userEvent from "@testing-library/user-event";
+import { useState } from "react";
+import { renderWithProviders, screen, within } from "__support__/ui";
+import type { SearchSidebarFilterComponent } from "metabase/search/types";
+import type { SearchSidebarFilterProps } from "./SidebarFilter";
+import { SidebarFilter } from "./SidebarFilter";
+
+const mockFilter: SearchSidebarFilterComponent = {
+  title: "Mock Filter",
+  iconName: "filter",
+  DisplayComponent: ({ value }) => (
+    <div data-testid="mock-display-component">
+      {!value || value.length === 0 ? "Display" : value}
+    </div>
+  ),
+  ContentComponent: ({ value, onChange }) => (
```

Comment:
We'll be testing the individual `Content` components for each filter, i.e. `TypeFilterContent` and `CreatedByFilterContent` etc